website/docs/troubleshooting.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "mongodbatlas"
+page_title: "Provider: MongoDB Atlas"
+sidebar_current: "docs-mongodbatlas-troubleshooting"
+description: |-
+  The MongoDB Atlas provider is used to interact with the resources supported by MongoDB Atlas. The provider needs to be configured with the proper credentials before it can be used.
+---
+
+# Troubleshooting
+
+The following are some of the common issues/errors encountered when using Terraform Provider for MongoDB Atlas :
+
+
+## Issue: Order of element objects in a TypeList attribute randomly changes everytime user runs `terraform plan`:
```

Comment:
Suggest adding a couple words to make it a complete sentence. It doesn't need to be, but I think it's easier to understand this way
```suggestion
## Issue: The order of element objects in a TypeList attribute randomly changes every time a user runs `terraform plan`:
```